python/django/security/django-no-csrf-token.yaml

```diff
@@ -0,0 +1,21 @@
+rules:
+  - id: django-no-csrf-token
+    patterns:
+      - pattern: "<form...>...</form>"
+      - pattern-not-inside: "<form...>...{% csrf_token %}...</form>"
+    message: Manually-created forms in django templates should specify a csrf_token to prevent CSRF attacks
+    languages: [generic]
+    severity: WARNING
+    metadata:
```

Comment:
I would include https://docs.djangoproject.com/en/4.2/howto/csrf/ from your PR description under `references` in metadata as well

Reply:
Added